src/model.py

```diff
@@ -71,6 +71,26 @@ def prettyprint(self) -> str:
         return f"RsaKey n={self.n} e={self.e}{d_part}"
 
 
+class EcdsaCurve:
+    """ Represents a ECDSA curve used by malware"""
+
+    def __init__(self, t: str, x: str, y: str) -> None:
```

Comment:
x and y should be ints (and probably should be cast to int before calling this constructor)

Reply:
Ok, changed to int. My only doubt is how they should be added to MISP. Document https://www.misp-project.org/objects.html#_crypto_material states clearly that is should be text. Should I convert them to str before add_attribute in to_misp() in EcdsaCurve class?

Reply:
good point! but I don't think it's necessary, they should be converted automatically.